README.md

```diff
@@ -4,13 +4,14 @@ This is a [client-go credential (exec) plugin](https://kubernetes.io/docs/refere
 
 ## Features
 
-* `convert-kubeconfig` command to converts kubeconfig with existing azure auth provider format to exec credential plugin format
-* device code login
-* non-interactive service principal login
-* non-interactive user principal login using [Resource owner login flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc)
-* non-interactive managed service identity login
-* AAD token will be cached locally for renewal in device code login and user principal login (ropc) flow. By default, it is saved in `~/.kube/cache/kubelogin/`
-* addresses https://github.com/kubernetes/kubernetes/issues/86410 to remove `spn:` prefix in `audience` claim, if necessary. (based on kubeconfig or commandline argument `--legacy`)
+- `convert-kubeconfig` command to converts kubeconfig with existing azure auth provider format to exec credential plugin format
+- [device code login](<#device-code-flow-(interactive)>)
+- [non-interactive service principal login](<#service-principal-login-flow-(non-interactive)>)
+- [non-interactive user principal login](<#user-principal-login-flow-(non-interactive)>) using [Resource owner login flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc)
+- [non-interactive managed service identity login](<#managed-service-identity-(non-interactive)>)
+- [non-interactive Azure CLI token login](<#azure-cli-token-login-(non-interactive)>)

```

Comment:
since we are adopting kubelogin for non managed Kubernetes on Azure as well, I think we should be clear that azure cli login will only work for AKS unless additional configuration is done?

Reply:
If you can tell me what other configuration would be needed, I can easily add that in. I didn't realize that it wouldn't work for non-managed K8s.

Reply:
I just played around with that, and I don't think it's possible. so AKS only :)